forage-android/src/main/java/com/joinforage/forage/android/network/EncryptionKeyService.kt

```diff
@@ -37,6 +37,7 @@ internal class EncryptionKeyService(
             .newBuilder()
             .addPathSegment(ForageConstants.PathSegment.ISO_SERVER)
             .addPathSegment(ForageConstants.PathSegment.ENCRYPTION_ALIAS)
+            .addPathSegment("")
```

Comment:
doing `addPathSegment("")` is not clear that we're adding a trailing slash. 

Would doing something like adding this extension work:

```kotlin
// in `OkHttpClientBuilder.kt` or new file: `HttpUrlExtensions.kt`
fun HttpUrl.Builder.addTrailingSlash(): HttpUrl.Builder {
    return this.addPathSegment("")
}

// in encryption key service

 return httpUrl
            .newBuilder()
            .addPathSegment(ForageConstants.PathSegment.ISO_SERVER)
            .addPathSegment(ForageConstants.PathSegment.ENCRYPTION_ALIAS)
            .addTrailingSlash()
            .build()
```

alternatively, can you add comments for each call to `addPathSegment("")`

Reply:
This is a pretty cool optimization I didn't know was possible :)

Reply:
[ed1f578](https://github.com/teamforage/forage-android-sdk/pull/121/commits/ed1f578f596f6653b1b3cf5d53c311ad5c1d9990)